Makefile

```diff
@@ -18,6 +18,10 @@ amazonlinux/% release/%: AMAZONLINUX_VERSION ?= $(call match_pattern_in_file,$(D
 amazonlinux/version:
 	@ echo $(AMAZONLINUX_VERSION)
 
+centos9/%: CENTOS8_VERSION ?= $(call match_pattern_in_file,$(DOCKERFILE_TOOLS),'centos:stream9','stream9@sha256:.*')
+centos9/version:
+	@ echo $(CENTOS8_VERSION)
+
```

Comment:
Oh that's funny. The var name actually doesn't matter, because it isn't used anywhere else. There's probably some way to refactor that, but probably not worth the squeeze. Fixed!